src/annotator/test/integration/anchoring-test.js

```diff
@@ -2,6 +2,7 @@
 // with various combinations of selector are anchored.
 
 import Guest from '../../guest';
+import { $imports as guestImports } from '../../guest';
```

Comment:
As a convention `$imports` in a test module refers to the imports for the module being tested. This however is an integration test so it doesn't have a 1:1 association with a particular module. Hence I've given this a more specific name.